### ISSUE TITLE
Vagrant Story ... freeze at loading screen after main menu [new game]   

### ISSUE THREAD

**Issue by ghost** [notaz/pcsx_rearmed#81]
Hi everyone,

i have a Problem with "Vagrant Story", it freezes after starting a new game at the loading screen.
The last thing i hear, is the ending of the sound that appears after pressing the button to confirm to start a new game. Can anyone tell me a soloution for this Problem?

System: RPI 3
OS: Recalbox 4.0.0 
PCSX Version: r22
BIOS: SCPH1001.BIN (confirmed md5sum)

 A lot of other games are running well.

Thanks!

**Comment by jjbriegas** [comment]
I have exactly the same problem, same RPI, same Recalbox, same bios also checksum confirmed...

It's an annoying issue, as the same .bin .cue Vagrant Story works well on PC PSX emulator, but on Recalbox it freezes at starting a new game

**Comment by rockaddicted** [comment]
SBI files missing ?
These versions require these files to work properly.

```
VAGRANT STORY
 PAL (ENGLISH) / SLES-02754
 PAL (FRENCH) / SLES-02755
 PAL (GERMAN) / SLES-02756
```

**Comment by jjbriegas** [comment]
Thanks a lot, I didn't knew that sbi files may be required. 
I'll let you know if it worked!

**Comment by John0Doe** [comment]
Where can I find those files? Thanks Am 01.09.2017 10:25 schrieb jjbriegas <notifications@github.com>:Thanks a lot, I didn't knew that sbi files may be required.
I'll let you know if it worked!

**Comment by jjbriegas** [comment]
I don't know if I can paste complete links here. I put it splitted, just join the parts

psx
data
center
.com
/sbifiles
.html

**Comment by John0Doe** [comment]
Commonly it is not allowed to post links, thanks anyway... Just delete this post in 2 minutes. ;DAm 01.09.2017 10:33 schrieb jjbriegas <notifications@github.com>:I don't know if I can paste complete links here. I put it splitted, just join the parts
psx
data
center
.com
/sbifiles
.html

**Comment by John0Doe** [comment]
I tried it with the sbi file... I put it in the same folder with the cue and bin file (psx) but I have the same freeze...Am 01.09.2017 10:11 schrieb rockaddicted <notifications@github.com>:SBI files missing ?
These versions require these files to work properly.
VAGRANT STORY
 PAL (ENGLISH) / SLES-02754
 PAL (FRENCH) / SLES-02755
 PAL (GERMAN) / SLES-02756

**Comment by rockaddicted** [comment]
`sbi/bin/cue` files have the same name ?

**Comment by jjbriegas** [comment]
maybe you need to edit .cue file to have the correct reference (name) for the .bin

**Comment by John0Doe** [comment]
Thanks for the hint, the name of the sbi file was even the default file name. I renamed it and now it works nicely! Thanks a lot! But why does especially this game such a file as an addition? Am 01.09.2017 12:15 schrieb jjbriegas <notifications@github.com>:maybe you need to edit .cue file to have the correct reference (name) for the .bin

**Comment by jjbriegas** [comment]
AFAIK, it seems that some games such as Vagrant, Crash Team Racing or Spyro II had a anticopy protection (back in time), and some emulators as the psx recalbox emulators need the sbi files to get through this anticopy protection.

thaks so much rockaddicted to point that hint out :D

**Comment by John0Doe** [comment]
Strangely CRT runs without an .sib File on my system! Maybe there are some different versions out there..?Am 01.09.2017 13:48 schrieb jjbriegas <notifications@github.com>:AFAIK, it seems that some games such as Vagrant, Crash Team Racing or Spyro II had a anticopy protection (back in time), and some emulators as the psx recalbox emulators need the sbi files to get through this anticopy protection.
thaks so much rockaddicted to point that hint out :D

**Comment by jjbriegas** [comment]
I tested it yesterday and now Vagrant Story and CTR both work fine.

For Vagrant the problem was clearly the .sbi missing, and for CTR I'm not sure if I renamed it on rpi and ruined the .cue reference, or if it needs sbi too, but having corrected both issues it works like a charm.

A last question: I'm aware of the possibility of converting a bin/cue heavy game onto light pbp format

Is it possible when games are not bin/cue, but bin/cue/sbi?
